docs/getting-started/release-notes/v2_11_0.md

```diff
@@ -47,17 +48,16 @@ Zowe Version 2.11.0 contains the bug fixes that are described in the following t
 
 #### Zowe CLI (Core)
 
-#### Zowe CLI Imperative Framework
-
-#### CICS Plug-in for Zowe CLI
-
-#### DB2 Plug-in for Zowe CLI
+- Fixed an issue in the Daemon server which prevents users on Windows with uppercase letters in their username from using the Daemon.
+- Add check for invalid block size when creating a sequential data set. ([#1439](https://github.com/zowe/zowe-cli/issues/1439))
+- Allowed `logout` operations with invalid and/or expired tokens. ([#1734](https://github.com/zowe/zowe-cli/pull/1734))
+- Prevented misleading `basePath error` when credentials are invalid. ([#1734](https://github.com/zowe/zowe-cli/pull/1734))
 
-#### MQ Plug-in for Zowe CLI
-
-#### IMS Plug-in for Zowe CLI
+#### Zowe CLI Imperative Framework
 
-### Zowe Explorer
+- Fixed merging of profile properties in `ProfileInfo.createSession`. ([#1008](https://github.com/zowe/imperative/issues/1008))
+- Allowed for multiple `auth logout` operations. ([#966](https://github.com/zowe/imperative/pull/996))
```

Comment:
@adam-wolfe not clear on this, this the bug allow multiple operations, or does the fix allow multiple operations?

Reply:
This would be a fix to allow a user to issue the `zowe auth logout apiml` command multiple times without failing.